src/components/structures/auth/Login.js

```diff
@@ -485,7 +529,7 @@ module.exports = React.createClass({
                onForgotPasswordClick={this.props.onForgotPasswordClick}
                loginIncorrect={this.state.loginIncorrect}
                serverConfig={this.props.serverConfig}
-               disableSubmit={this.isBusy()}
+               disableSubmit={this.isBusy() || !this.state.serverIsAlive}
```

Comment:
Hmm - if it fails to fetch the flows then it will still let you submit the form - this may not be the right decision but it means the liveness check is inconsistent with this. If we are going to disable the submit button until the liveness check succeeds, I think we need a 'retry' button, otherwise the user has no obvious way to proceed.

Reply:
I suppose it's probably easier to just enable the button and re-check liveliness then.